lib/rules/color-hex-case/__tests__/index.js

```diff
@@ -164,3 +164,181 @@ testRule(
 		],
 	}),
 );
+
+testRule({
```

Comment:
I think we got our wires crossed with the indents. Let's indent the template literals as normal blocks, like so:

```js
testRule({
	ruleName,
	config: ['upper'],
	syntax: 'css-in-js',
	fix: true,
	accept: [
		{
			code: `
				export const s = styled.a({
				    color: "#AAA",
				});
			`,
		},
	],

	reject: [
		{
			code: `
				export const s = styled.a({
				    color: "#aBABAB",
				});
			`,
			fixed: `
				export const s = styled.a({
					color: "#ABABAB",
				});
			`,
			message: messages.expected('#aBABAB', '#ABABAB'),
			line: 3,
			column: 15,
		},
	],
});
```

Reply:
Ah, apologies! Github's 8-spaces-per-tab made it look like you were trying to align it with the backtick, not just trying to indent it by one line. I'll take care of this today.